CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ No changes to highlight.
 - Fix `gr.Slider` `release` event not triggering on mobile by [@space-nuko](https://github.com/space-nuko) in [PR 4098](https://github.com/gradio-app/gradio/pull/4098) 
 - Removes extraneous `State` component info from the `/info` route by [@abidlabs](https://github.com/freddyaboulton) in [PR 4107](https://github.com/gradio-app/gradio/pull/4107)
 - Make .then() work even if first event fails by [@aliabid94](https://github.com/aliabid94) in [PR 4115](https://github.com/gradio-app/gradio/pull/4115).
+- Scroll event performance by [@shavit](https://github.com/shavit) in [PR 4158](https://github.com/gradio-app/gradio/pull/4158).
```

Comment:
This change does not require a changelog entry.

Reply:
Alright.